kernels/portable/cpu/util/broadcast_util.cpp

```diff
@@ -19,16 +19,14 @@ namespace executor {
 using Tensor = exec_aten::Tensor;
 using ScalarType = exec_aten::ScalarType;
 
-void free_broadcast_tensor(const Tensor& broadcast_tensor) {
-  free((void*)broadcast_tensor.const_data_ptr());
-  free((void*)broadcast_tensor.sizes().data());
-  free((void*)broadcast_tensor.dim_order().data());
-  free((void*)broadcast_tensor.strides().data());
-  free(broadcast_tensor.unsafeGetTensorImpl());
+void free_tensor(const Tensor& tensor) {
+  free((void*)tensor.const_data_ptr());
+  free((void*)tensor.sizes().data());
+  free((void*)tensor.dim_order().data());
+  free((void*)tensor.strides().data());
+  free(tensor.unsafeGetTensorImpl());
 }
 
-namespace {
-
```

Comment:
It's interesting that malloc is used in this function but I don't see how the malloced pointer is freed. @SS-JIA @manuelcandales , could you provide more context that I miss? cc @dbort

Reply:
There's a corresponding function called `free_tensor` that when passed in the tensor created via this function free's it up.

Reply:
@tarun292 These functions are not being used in the portable kernels. I believe you introduced these utils back in January 2023, and maybe they were being used at that time, but not anymore. They should be moved out of the portable lib utils.